src/main/java/com/dumbdogdiner/stickyapi/bukkit/book/commonmarkextensions/MCBlockParser.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.book.commonmarkextensions;
+
+import org.commonmark.node.Block;
+import org.commonmark.parser.block.AbstractBlockParser;
+import org.commonmark.parser.block.BlockContinue;
+import org.commonmark.parser.block.ParserState;
+
+public class MCBlockParser extends AbstractBlockParser {
```

Comment:
Change this to `BlockParser`

```suggestion
public class MCBlockParser extends AbstractBlockParser {
```